src/components/MaterialUI/DataDisplay/Tooltip.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useRef } from "react";
+import MuiTooltip from "@material-ui/core/Tooltip";
+import { makeStyles } from "@material-ui/core/styles";
+import classNames from "classnames";
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    maxWidth: props => props.maxWidth,
+    display: "inline-block"
+  },
+  noWrap: {
+    whiteSpace: "nowrap",
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  },
+  tooltip: {
+    maxWidth: "100%",
+    "& p": {
+      fontSize: theme.typography.fontSize
+    }
+  }
+}));
+
+const Tooltip = ({ value, maxWidth = 0 }) => {
+  const classes = useStyles({ maxWidth });
+
+  const ref = useRef(null);
+
+  const [isDisplayed, setIsDisplayed] = React.useState(false);
+
+
+  useEffect(() => {
+    setIsDisplayed(ref.current.clientWidth >= maxWidth);
+  }, [maxWidth]);
```

Comment:
Add "ref" between squared brackets and add the eslint comment about "setIsDisplayed"

Reply:
done